src/read_vegparam.c

```diff
@@ -239,16 +239,23 @@ veg_con_struct *read_vegparam(FILE *vegparam,
         nrerror(ErrStr);
       }
 
+      for ( j = 0; j < 12; j++ ) {
```

Comment:
But there are several other existing loops over j from 0 to 12 in this function.  Changing only this one will be inconsistent with the rest of the function...  And changing all of them, well, is it my job to overhaul all of the constants just because I'm making a rather minor change to the function?